jobs/aws/eks-anywhere-build-tooling/harbor-helm-presubmits.yaml

```diff
@@ -34,7 +34,7 @@ presubmits:
         - bash
         - -c
         - >
-          if $(make check-project-path-exists); then make build -C $PROJECT_PATH; fi
+          if $(make check-project-path-exists); then make helm/push -C $PROJECT_PATH; fi
```

Comment:
We don't want to push on a presubmit. I think this was fine. `helm/build` maybe, but `build` should include that.

Reply:
Yeh, and the lint job is picky on make targets